node_cli/core/resources.py

```diff
@@ -61,6 +65,21 @@ def get_resource_allocation_info():
         return None
 
 
+def get_schain_allocation_info():
```

Comment:
no need to read schain allocation again, just use: `schain_allocation_data = safe_load_yml(ALLOCATION_FILEPATH)` from `compose_resource_allocation_config` and pass to your function